### DISCUSSION TITLE
Canary Rollback

### DISCUSSION THREAD

**Discussion by dlennon-ie** [argoproj/argo-rollouts#2059] [Q&A]
Hi,
Trying to figure out how to abort a canary rollout and return the rollout to a healthy status.
Whenever I abort a canary rollout the traffic is moved off successfully however the Rollout status is set to degraded.
What is the correct approach to rollback a canary release?

**Comment by kostis-codefresh** [comment]
Hello

The Rollout is set to degraded because the new release was not deployed

The old version is still running and is perfectly healthy. There is nothing else to do. You are expected to fix the error and redeploy in order to get a "healthy" version.

Not also that ArgoCD health status and Argo Rollouts status is not 1-1 mapping. It is manually mapped via the health check https://github.com/argoproj/argo-cd/blob/master/resource_customizations/argoproj.io/Rollout/health.lua
